### DISCUSSION TITLE
Condition Listing Data of Resource

### DISCUSSION THREAD

**Discussion by burhanahmed92** [filamentphp/filament#2905] [Help] [answered]
I have created a Model Employee and there is a column Company ID used as foreign id. So I created an Employee Resource to generate a CRUD easily. 
But the issue is that I want to display only  Employee that logged in user is associated with the attached company. 
Is there a way to do that?

**Comment by studiowizjo** [comment, marked as answer]
Add to your `ListEmployees.php` page:

```php
protected function getTableQuery(): Builder
{
  return Employee::where('company_id', auth()->user()->company_id);
}
```

**Reply by danharrin** [reply]
Should probably use `getEloquentQuery()` on the resource instead, so access to the Edit page is also controlled

https://filamentphp.com/docs/2.x/admin/resources/getting-started#simple-multi-tenancy-from-scratch
